src/js/view/controls/components/chapters.mixin.js

```diff
@@ -33,8 +33,10 @@ const ChaptersMixin = {
     chaptersLoaded: function (evt) {
         const data = srt(evt.responseText);
         if (Array.isArray(data)) {
-            data.forEach((obj) => this.addCue(obj));
-            this.drawCues();
+            // Add chapter cues directly to model which will trigger addCue()
+            const existingCues = this._model.get('cues');
+            const newCues = existingCues ? existingCues.concat(data) : data;
```

Comment:
Guarantee that model 'cues' is always an array so you don't have to check if existingCues is defined (see above)